### ISSUE TITLE
Generator does not support C++, Rust, Objective-C and others

### ISSUE THREAD

**Issue by mloskot** [swagger-api/swagger-codegen#11052]
##### Description

What is the status of C++, Rust, Objective-C and som eother languages listed as **supported** in the current [README.md](https://github.com/swagger-api/swagger-codegen/blob/4a445fc0d6cf85f0e6b1aaf060681ca1ad833232/README.md) but actually not supported by the currently latest Swagger Codegen JAR.

The current [README.md](https://github.com/swagger-api/swagger-codegen/blob/4a445fc0d6cf85f0e6b1aaf060681ca1ad833232/README.md) says:

 >Currently, the following languages/frameworks are supported:
> API clients: ActionScript, Ada, Apex, Bash, C# (.net 2.0, 3.5 or later), C++ (cpprest, Qt5, Tizen), Clojure, Dart, Elixir, Elm, Eiffel, Erlang, Go, Groovy, Haskell (http-client, Servant), Java (Jersey1.x, Jersey2.x, OkHttp, Retrofit1.x, Retrofit2.x, Feign, RestTemplate, RESTEasy, Vertx, Google API Client Library for Java, Rest-assured), Kotlin, Lua, Node.js (ES5, ES6, AngularJS with Google Closure Compiler annotations) Objective-C, Perl, PHP, PowerShell, Python, R, Ruby, Rust (rust, rust-server), Scala (akka, http4s, swagger-async-httpclient), Swift (2.x, 3.x, 4.x, 5.x), Typescript (Angular1.x, Angular2.x, Fetch, jQuery, Node)

##### Swagger-codegen version

3.0.26

##### Related issues/PRs

```
java -jar .\swagger-codegen-cli-3.0.26.jar config-help -l cpprest
java -jar .\swagger-codegen-cli-3.0.26.jar config-help -l rust
java -jar .\swagger-codegen-cli-3.0.26.jar config-help -l objective-c
```

**Comment by mloskot** [comment]
The 3.0.33 is still missing those languages :-(

**Comment by DnOberon** [comment]
Yeah...wasn't enthused to find out they're no longer supported the hard way.

**Comment by bnheise** [comment]
Can someone update the readme? I was also confused by this.

**Comment by CJCombrink** [comment]
Is C++ generation something that is planned for version 3?
